python/ray/util/sgd/v2/worker_group.py

```diff
@@ -22,6 +25,26 @@ def __execute(self, func: Callable[..., T], *args, **kwargs) -> T:
         return func(*args, **kwargs)
 
 
+@dataclass
+class WorkerMetadata:
+    """Metadata for each worker/actor.
+
+    This information is expected to stay the same throughout the lifetime of
+    actor.
+    """
+    node_id: str  # ID of the node this worker is on.
+    node_ip: str  # IP address of the node this worker is on.
+    hostname: str  # Hostname that this worker is on.
+    gpu_ids: Optional[List[int]]  # List of CUDA IDs available to this worker.
```

Comment:
Move these docs up to class doc? Like [CheckpointStrategy](https://github.com/ray-project/ray/blob/d081ee9d87d3176005263687a7ab315a9b8f2f69/python/ray/util/sgd/v2/checkpoint.py).

Reply:
Done